src/utils/authentication.js

```diff
@@ -0,0 +1,20 @@
+import jwt from 'jsonwebtoken';
+
+export const generateToken = (payload, expiresIn = '24h') => jwt.sign(payload, process.env.JWT_SECRET, { expiresIn });
+
+export const shuffleToken = (token) => {
+  token.split('').reverse().join('');
+};
+
+export const verifyToken = (req, res, input) => {
```

Comment:
The verify token method should have only the `token` as a parameter. We might want to use this method in other parts of the codebase that don't use the req, res objects. Also, using `input` as a parameter name is not clear. A more descriptive variable name like `token` is more descriptive.